webserver/views/user.py

```diff
@@ -10,8 +10,9 @@
 
 @user_bp.route("/<musicbrainz_id>")
 def profile(musicbrainz_id):
-    if current_user.is_authenticated and \
-       current_user.musicbrainz_id == musicbrainz_id:
```

Comment:
Old version breaks when username in the URL doesn't have letters in the same case as the original username. For example, when logged in I see controls at `/user/Gentlecat`, but not at `/user/gentlecat`.